### DISCUSSION TITLE
[v0.48.0] Redaction Processor

### DISCUSSION THREAD

**Discussion by Harnoor-se7en** [open-telemetry/opentelemetry-collector-contrib#9246] [Q&A]
Hi 
I was testing the redaction processor in order to mask values of span attributes. The image used - [https://hub.docker.com/layers/opentelemetry-collector-contrib/otel/opentelemetry-collector-contrib/0.48.0/images/sha256-1a9746ab7c28e25468a60b16eaa3b11a64ce7dd168424a4aab0d6dbb6df3a329?context=explore](url)

But upon using the above image I am getting this error:

> cannot unmarshal the configuration: unknown processors type "redaction" for "redaction"

Wanted to confirm if this processor is part of the latest release and if it's not can we please update its read me file with the appropriate status like that of **Delta to Rate Processor** -
Status: under development; Not recommended for production usage.

Thanks

**Comment by jpkrohling** [comment]
Apparently, it's not in the contrib's manifest: https://github.com/open-telemetry/opentelemetry-collector-releases/blob/main/distributions/otelcol-contrib/manifest.yaml

cc @leonsp-ai @dmitryax @mx-psi

**Reply by dmitryax** [reply]
@leonsp-ai is the redaction processor ready to be used? If so can you please add it to the manifest ^?

**Reply by Harnoor-se7en** [reply]
@leonsp-ai ^^ (Sorry for re-tagging)
